packages/api/api-generator/package.json

```diff
@@ -0,0 +1,71 @@
+{
+    "name": "@comet/api-generator",
+    "version": "7.13.0",
+    "repository": {
+        "directory": "packages/api/api-generator",
+        "type": "git",
+        "url": "https://github.com/vivid-planet/comet"
+    },
+    "license": "BSD-2-Clause",
+    "main": "lib/index.js",
+    "typings": "lib/index.d.ts",
+    "bin": {
+        "comet-api-generator": "bin/api-generator.js"
+    },
+    "files": [
+        "bin/**/*.js",
+        "lib/*"
+    ],
+    "scripts": {
+        "build": "$npm_execpath run clean && tsc -p tsconfig.build.json",
+        "clean": "rimraf lib",
+        "dev": "tsc --watch --preserveWatchOutput -p tsconfig.build.json",
+        "lint": "run-p lint:prettier lint:eslint lint:tsc",
+        "lint:eslint": "eslint --max-warnings 0 src/ **/*.json --no-warn-ignored",
+        "lint:prettier": "npx prettier --check './**/*.{js,json,md,yml,yaml}'",
+        "lint:tsc": "tsc",
+        "test": "NODE_OPTIONS=--experimental-vm-modules npx jest",
+        "test:watch": "jest --watch"
+    },
+    "dependencies": {
+        "@comet/cms-api": "workspace:*",
+        "commander": "^9.5.0",
+        "pluralize": "^8.0.0",
+        "prettier": "^3.5.0",
+        "rimraf": "^6.0.1",
+        "ts-morph": "^16.0.0",
+        "uuid": "^11.0.5"
+    },
+    "devDependencies": {
+        "@comet/eslint-config": "workspace:^7.13.0",
+        "@kubernetes/client-node": "^1.0.0",
+        "@mikro-orm/cli": "^6.4.5",
+        "@mikro-orm/core": "^6.4.5",
+        "@mikro-orm/postgresql": "^6.4.5",
+        "@nestjs/common": "^10.4.15",
+        "@nestjs/core": "^10.4.15",
+        "@nestjs/graphql": "^12.2.2",
```

Comment:
Do we need all these dependencies?

Reply:
in my opinion, we need them all - knip is also that opinion. 

But we maybe should move them from dev dependencies into dependencies

Reply:
i would say, following dependencies should go into dependencies:
```
        "@kubernetes/client-node": "^1.0.0",
        "@mikro-orm/cli": "^6.4.5",
        "@mikro-orm/core": "^6.4.5",
        "@mikro-orm/postgresql": "^6.4.5",
        "@nestjs/common": "^10.4.15",
        "@nestjs/core": "^10.4.15",
        "@nestjs/graphql": "^12.2.2",
        "class-transformer": "^0.5.1",
        "class-validator": "^0.14.1",
        "graphql": "^16.9.0",
        "reflect-metadata": "^0.2.2",
```

Reply:
moved them.

Reply:
refactored dependencies / dev dependencies / peer dependencies.